cluster-provision/centos8/Dockerfile

```diff
@@ -23,7 +23,7 @@ COPY scripts/download_box.sh /
 
 RUN echo "Centos8 version $centos_version"
 
-ENV CENTOS_URL https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-Vagrant-8-$centos_version.x86_64.vagrant-libvirt.box
+ENV CENTOS_URL https://kojihub.stream.centos.org/kojifiles/packages/CentOS-Stream-Vagrant/8/$centos_version/images/CentOS-Stream-Vagrant-8-$centos_version.x86_64.vagrant-virtualbox.box
```

Comment:
I think we would need the libvirt.box here instead of the box built for virtualbox?

Reply:
Good catch, thanks!